game/plugin/skills/mining/build.gradle

```diff
@@ -2,12 +2,12 @@ plugin {
     name = "mining-skill"
```

Comment:
Can you remove the change to the mining build.gradle from this plugin please? The indentation does look off but each PR should ideally be a set of related changes.

Reply:
Yeah, sorry. Fighting idea.